src/components/Notifications/NotificationTile.tsx

```diff
@@ -0,0 +1,139 @@
+  const { palette, isLight } = usePalette()
+
+  const sender = props.notification.content.data?.sender
+  const association = props.notification.content.data?.association
+  const object = props.notification.content.data?.object
+  const linkUrl = props.notification.content.data?.linkUrl
+  const identifier = props.notification.identifier
+  const isRead = props.isRead ?? props.notification.isRead
+
+  return (
+    <TouchableRipple onClick={props.onPress} showRipple={props.showRipple}>
+      <SharedElement id={props.notification.identifier}>
+        <View style={{ paddingVertical: 10, paddingHorizontal: 28 }}>
+          <View
+            style={{
+              flexDirection: "row",
+              flex: 1,
+            }}
+          >
+            <Image
+              source={{
+                uri: "https://images.unsplash.com/photo-1560174038-da43ac74f01b?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2057&q=80",
+              }}
+              style={{
+                width: 48,
+                height: 48,
+                resizeMode: "cover",
+
+                borderRadius: 24,
+              }}
+            />
+            <View
+              style={{
+                marginLeft: 8,
+                flex: 1,
+                justifyContent: "space-around",
+              }}
+            >
+              <BodyText
+                style={{
+                  fontSize: 16,
+                  fontWeight: "700",
+                  color: isLight ? palette.purpleVariant : "#fff",
+                }}
+              >
+                {sender}
+                {association ? ` - ${association}` : ""}
+              </BodyText>
+              <BodyText
+                style={{
+                  fontSize: 12,
+                  fontWeight: "700",
+                  color: isLight ? palette.purpleVariant : "#fff",
+                }}
+              >
+                {object}
+              </BodyText>
+            </View>
+            <View
+              style={{
+                justifyContent: "space-between",
+              }}
+            >
+              <Pressable
+                onPress={() => {
+                  notificationCentre.removeNotificationFromStorage(identifier)
+                  if (!isRead) {
+                    notificationEventEmitter.emit("badge-change")
+                  }
+                  if (props.onCancel) {
+                    props.onCancel()
+                  }
+                }}
+              >
+                <Icon source={deleteSvg} style={{ alignSelf: "flex-end" }} />
+              </Pressable>
+              {!isRead && (
+                <BodyText
+                  style={{
+                    fontSize: 12,
+                    fontWeight: "700",
+                    color: isLight ? palette.purpleVariant : "#fff",
+                  }}
+                >
+                  New!!
+                </BodyText>
```

Comment:
non mi piace tantissimo sta parte esteticamente

la scritta New!! è un po' tanto e contemporaneamente non risalta, sarebbe carino secondo me avere un singolo banale pallino dell'arancione accent centrato, e per eliminare la notifica quanto sarebbe difficile implementare lo "swipe left to delete" classico di iOS? credo che questo sia il giusto punto di inizio per una cosa del genere https://docs.swmansion.com/react-native-gesture-handler/docs/api/components/swipeable/

ma volendo possiamo lasciarla così come proof of concept e se ti piace questa cosa dello swipe in una task apposta in un altro momento, tanto sicuramente le notifiche dovranno essere modificate nel corso dello sviluppo anyway

Reply:
Volendo sono riuscito a implementare lo swipe to left però non so come fare il design, ad esempio tutto rosso (e quale rosso) con l'icona di un bidone? Magari adesso faccio il commit senza questa cosa grafica e domani chiediamo al team di design di definire bene il tutto oppure mi dici te.